packages/authorization-process/src/services/authorizationService.ts

```diff
@@ -416,3 +552,12 @@ const assertSecurityUser = async (
     throw securityUserNotFound(requesterUserId, userId);
   }
 };
+
+const assertOrganizationIsPurposeConsumer = (
```

Comment:
It's best if we keep all the asserts in `validators`

Reply:
Updated